### DISCUSSION TITLE
Can molstar be implemented in a vue project?

### DISCUSSION THREAD

**Discussion by RelaxRick** [molstar/molstar#424] [Q&A] [answered]
Can molstar be implemented in a vue project?

**Comment by dsehnal** [comment, marked as answer]
Yes, you can use only the rendering capabilities of Mol* and write your UI in Vue. All that Mol* needs is that you pass it a DIV and CANVAS: https://github.com/molstar/molstar/blob/master/src/mol-plugin/context.ts#L188 .

If you want to use the UI that comes along with Mol*, you can just mount a React component inside VUE too I believe.

**Reply by RelaxRick** [reply]
thanks for your answer， i try

**Comment by kuangnannan** [comment]
Can you share the code implemented in Vue？
